src/ModelContextProtocol.AspNetCore/McpEndpointRouteBuilderExtensions.cs

```diff
@@ -50,6 +51,10 @@ public static IEndpointConventionBuilder MapMcp(this IEndpointRouteBuilder endpo
 
             try
             {
+                // Make sure we disable all response buffering for SSE
+                context.Response.Headers.ContentEncoding = "identity";
+                context.Features.GetRequiredFeature<IHttpResponseBodyFeature>().DisableBuffering();
```

Comment:
We want to fail if the IHttpRespondBodyFeatureService isn't present? Or it doesn't really matter because it always will be?

Reply:
It's always there. It's what also [backs](https://github.com/dotnet/aspnetcore/blob/bd4a06387dec0939c9afd7beb1f547fd93ad9f6c/src/Http/Http/src/Internal/DefaultHttpResponse.cs#L69) `HttpResponse.Body`. Basically, nothing would work without it.